aws_policy_equivalence.go

```diff
@@ -396,6 +400,10 @@ func newAWSPrincipalStringSet(members interface{}) awsPrincipalStringSet {
 }
 
 func (actions awsStringSet) equals(other awsStringSet) bool {
+	if actions == nil || other == nil {
```

Comment:
Need this or else any two invalid `Resource` statements (or `NotResource`, `Action`, `NotAction` etc.) would be equal.